src/breakpoints.jl

```diff
@@ -281,3 +281,14 @@ macro breakpoint(call_expr, args...)
         end
     end
 end
+
+const __BREAKPOINT_MARKER__ = nothing
+
+"""
+    @bp
+
+Insert a breakpoint at a location in the source code.
+"""
+macro bp()
```

Comment:
Add an optional condition?

Reply:
I thought about this but is `@bp i > 0` much easier than `i > 0 && @bp`?

Reply:
For example

```jl
julia> function f(x)
      println("hello")
      x > 5 && @bp
      return x
  end;

julia> @interpret f(2)
hello
2

julia> @interpret f(7)
hello
(Frame for f(x) in Main at REPL[41]:2
⋮
  2 3  │   %2 = (>)(x, 5)
  3 3  └──      goto #3 if not %2
b 4 3  2 ─      nothing
  5 3  └──      goto #3
  6 4  3 ┄      return x
x = 7, breakpoint(f(x) in Main at REPL[41]:2, line 3))
```

Reply:
That works great, good point! Just need to make sure it's documented; obviously I didn't think of it, so others may not either.